src/Microsoft.Azure.SignalR.Common/Microsoft.Azure.SignalR.Common.csproj

```diff
@@ -17,10 +17,13 @@
     <PackageReference Include="System.IO.Pipelines" Version="$(SystemIOPipelinesPackageVersion)" />
   </ItemGroup>
 
+  <!-- Note to not reference Microsoft.AspNetCore.Http.Connections.Client as it has breaking changes from .NETSTANDARD2.0 to NETCOREAPP3.0-->
   <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0'">
-    <PackageReference Include="Microsoft.AspNetCore.Http.Connections.Client" Version="$(MicrosoftAspNetCoreHttpConnectionsClientPackage3_0Version)" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Connections.Common" Version="$(MicrosoftAspNetCoreHttpConnectionsClientPackage3_0Version)" />
```

Comment:
What's in here?

Reply:
I think you just need https://www.nuget.org/packages/Microsoft.AspNetCore.Connections.Abstractions